_shared/project/conf/supervisord.conf

```diff
@@ -3,29 +3,14 @@ nodaemon=true
 environment=PYTHONUNBUFFERED="1"
 logfile=/dev/null
 logfile_maxbytes=0
-{% if cookiecutter.get("_directory") == "pyapp" %}
 
-[program:app]
-command=python3 -m {{ cookiecutter.package_name }}
-stdout_logfile=NONE
-stderr_logfile=NONE
-stdout_events_enabled=true
-stderr_events_enabled=true
-{% endif %}
-{% if cookiecutter.get("_directory") == "pyramid-app" %}
-
-[program:web]
-command=newrelic-admin run-program gunicorn --paste conf/production.ini --bind 0.0.0.0:{{ cookiecutter.port }}
-stdout_logfile=NONE
-stderr_logfile=NONE
-stdout_events_enabled=true
-stderr_events_enabled=true
-{% endif %}
-{% if include_exists("conf/supervisord.conf") %}
-
-{{ include("conf/supervisord.conf") -}}
-{% endif %}
+{% for program_name, program_data in supervisor_settings().programs.items() %}
+[program:{{ program_name }}]
+{% for setting_name, setting_value in program_data.settings.items() %}
+{{ setting_name }}={{ setting_value }}
+{% endfor %}
```

Comment:
This for-loop has to be duplicated because with cookiecutter there's no way to share template or macro code between files. But at least the Python `supervisor_settings()` method has done most of the work so the duplicated for-loop is trivial.